docs/development/extensions-core/stats.md

```diff
@@ -35,9 +35,12 @@ The American Statistician, 37 (1983) pp. 242--247.
 
 variance = variance1 + variance2 + n/(m*(m+n)) * pow(((m/n)*t1 - t2),2)
 
-where: - variance is sum(x-avg^2) (this is actually n times the variance)
-and is updated at every step. - n is the count of elements in chunk1 - m is
-the count of elements in chunk2 - t1 = sum of elements in chunk1, t2 =
+where: 
+ - variance is sum(x-avg^2) (this is actually n times the variance)
+and is updated at every step. 
+ - n is the count of elements in chunk1 
+ - m is the count of elements in chunk2 
+ - t1 = sum of elements in chunk1, t2 =
```

Comment:
Since all parameters are formatted as list item above, I think It's better to put `t2` in a separate list item too.